watchdog/alerting.go

```diff
@@ -49,10 +59,14 @@ func handleAlertsToTrigger(endpoint *core.Endpoint, result *core.Result, alertin
 			if err != nil {
 				log.Printf("[watchdog][handleAlertsToTrigger] Failed to send an alert for endpoint=%s: %s", endpoint.Name, err.Error())
 			} else {
-				endpointAlert.Triggered = true
+				// Mark initial alert as triggered and update last reminder time
+				if sendInitialAlert {
+					endpointAlert.Triggered = true
+				}
+				endpoint.LastReminderSent = time.Now()
 			}
 		} else {
-			log.Printf("[watchdog][handleAlertsToResolve] Not sending alert of type=%s despite being TRIGGERED, because the provider wasn't configured properly", endpointAlert.Type)
+			log.Printf("[watchdog][handleAlertsToResolve] Not sending alert of type=%s despite being due, because the provider wasn't configured properly", endpointAlert.Type)
```

Comment:
I left it as `TRIGGERED` specifically because the opposite is `RESOLVED` - makes it easier to look through the logs 😅